crates/aptos-rosetta/src/types/objects.rs

```diff
@@ -720,9 +720,9 @@ fn parse_operations_from_write_set(
                 if id == &withdraw_events_identifier() {
                     serde_json::from_value::<CoinEventId>(value.clone()).unwrap();
                     if let Ok(event) = serde_json::from_value::<CoinEventId>(value.clone()) {
-                        let withdraw_event = EventKey::new_from_address(
-                            &event.guid.guid.id.addr,
+                        let withdraw_event = EventKey::new(
                             event.guid.guid.id.creation_num.0,
+                            event.guid.guid.id.addr,
```

Comment:
this looks awful

Reply:
and it is bad, thanks for catching that... there are not tests here, but I know they should just be guid.id.